runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -486,6 +489,15 @@ public void onChatMessage(ChatMessage event)
 			return;
 		}
 
+		final Matcher pickpocketMatcher = pickpocketingRegex.matcher(message);
+		if (config.trackPickpockets() && pickpocketMatcher.matches())
+		{
+			String pickpocketTarget = WordUtils.capitalize(pickpocketMatcher.group("target"));
+			eventType = pickpocketTarget + " (Pickpocket)";
```

Comment:
I would recommend adding e.g. `private static final String PICKPOCKET_EVENT = " (Pickpocket)"` and using this here and in line 542.

Reply:
Mmm, that's a good idea. I had a const at first but I took it out because I hadn't thought to use it in both places